### PR TITLE
support reading mdio config from APCI tables

### PR DESCRIPTION
**Pull request by msiddare**
Current mdio-mux does not support reading configuration
from ACPI tables

cisco-8000 platform configures mdio phy config through ACPI
tables.

Added support int he mdio-mux driver to read from ACPI.

dump of acpidump attached
[ssdt1.zip](https://github.com/Azure/sonic-linux-kernel/files/6657907/ssdt1.zip)

Signed-off-by: Madhava Reddy Siddareddygari <msiddare@cisco.com>

### CODE REVIEW

File: patch/cisco-mdio-mux-support-acpi.patch [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+Date: Tue, 15 Jun 2021 11:32:49 -0700
+Subject: [PATCH] support reading mdio config from APCI tables
+
+Current mdio-mux does not support reading configuration
+from ACPI tables.
+
+cisco-8000 platform configures mdio phy config through ACPI
+tables.
+
+Added support int he mdio-mux driver to read from ACPI
```
Comment (lguohan) [inline patch/cisco-mdio-mux-support-acpi.patch:12]: int he -> in the

File: patch/cisco-mdio-mux-support-acpi.patch [hunk 2]
```diff
@@ -0,0 +1,85 @@
+From 3f82482489e986fc3b0a38163902842d0c4fa2f2 Mon Sep 17 00:00:00 2001
+From: Madhava Reddy Siddareddygari <msiddare@cisco.com>
+Date: Tue, 15 Jun 2021 11:32:49 -0700
+Subject: [PATCH] support reading mdio config from APCI tables
```
Comment (prgeor) [inline patch/cisco-mdio-mux-support-acpi.patch:4]: typo "ACPI"

File: patch/cisco-mdio-mux-support-acpi.patch (hunk 1 above)
Comment (msiddare) [reply patch/cisco-mdio-mux-support-acpi.patch]: Fixed Typo

File: patch/cisco-mdio-mux-support-acpi.patch (hunk 2 above)
Comment (msiddare) [reply patch/cisco-mdio-mux-support-acpi.patch]: Fixed Typo

**Review by lguohan — APPROVED** [review]

**Pull request merged**